src/the-bureau/index.html

```diff
@@ -184,8 +186,8 @@ <h1 class="h2">CFPB leadership</h1>
                              alt="Director Richard Cordray">
                     </div>
                     <div class="media_body">
-                        <h1 class="h6">Director</h1>
-                        <h2 class="bureau-bio_name">Richard Cordray</h2>
+                        <span class="h6">Director</span>
```

Comment:
I'm totally willing to change it for consistency one way or the other, but my thinking was that the bureau-structure page was more focused on titles and office names and folks would navigate that page by office.The about and bureau pages are more about the individuals.

Here's how the headings structure looks for bureau-structure.
![image](https://cloud.githubusercontent.com/assets/1860176/9803022/c89c1d8e-57ec-11e5-93fe-e5b6490c1ddd.png)